backend/grant-service/api/src/main/kotlin/Grants.kt

```diff
@@ -822,6 +854,26 @@ object Grants : CallDescriptionContainer("grant") {
         }
     }
 
+    val findAffiliations = call<FindAffiliationsRequest, FindAffiliationsResponse, CommonErrorMessage>("FindAffiliations") {
```

Comment:
Name should match perfectly but has an upper-case F in the string. Renaming to `retrieve` would also be more in line with (new) conventions.
```suggestion
    val retrieveAffiliations = call<GrantsRetrieveAffiliationsRequest, GrantsRetrieveAffiliationsResponse, CommonErrorMessage>("retrieveAffiliations") {
```